ui/tests/integration/components/totp/page/totp-list-item-test.js

```diff
@@ -0,0 +1,46 @@
+import EmberObject from '@ember/object';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | totp-list-item', function (hooks) {
```

Comment:
it seems like we already test all of this functionality in the acceptance test, i think a component test for key-create may be more useful to confirm things like the getter/setter you added update the model and submit the expected payload.